python/private/toolchains_repo.bzl

```diff
@@ -194,6 +202,13 @@ def py_binary(name, **kwargs):
         **kwargs
     )
 
+def py_entry_point_binary(name, **kwargs):
```

Comment:
Should we add docs here?  Any reason why we are using **kwargs?  I am not a huge fan of it :(

Reply:
We shouldn't do it here IMO, because `py_binary` has too many of them and I would not like to duplicate docs for them.